TUM-Dev/NavigaTUM
# [Cal-11] Postgres

Currently we use SQlite to store our data.
This approach has a few downsides:
- SQLite manages state in one file (i.e. writes lock the entire database)
- Since all Staging branches share this state, they need to be scheduled on the same node, making the system unnecessarily brittle

This PR migrates this datastore to Postgres, which should be a better long-term solution to this problem

## Proposed Changes (include Screenshots if possible)

- Migration from sqlite to postgres as a datastore for the Calendar

## How to test this PR

- code review

## How has this been tested?

1. Launch `docker run -p 5432:5432 -e POSTGRES_PASSWORD=password postgres`
2. Lanch the calendar
3. See it continuing to scrape

## Checklist:
